### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.media.models.api_error.ApiError
  - azure.mgmt.media.models.api_error.ApiErrorException
  fullName: azure.mgmt.media.models.api_error
  langs:
  - python
  module: azure.mgmt.media.models.api_error
  name: api_error
  source:
    id: api_error
    path: azure-mgmt-media\azure\mgmt\media\models\api_error.py
    remote:
      branch: master
      path: azure-mgmt-media\azure\mgmt\media\models\api_error.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.media.models.api_error
references:
- fullName: azure.mgmt.media.models.api_error.ApiError
  isExternal: false
  name: ApiError
  parent: azure.mgmt.media.models.api_error
  uid: azure.mgmt.media.models.api_error.ApiError
- fullName: azure.mgmt.media.models.api_error.ApiErrorException
  isExternal: false
  name: ApiErrorException
  parent: azure.mgmt.media.models.api_error
  uid: azure.mgmt.media.models.api_error.ApiErrorException
